README.md

```diff
@@ -123,6 +146,107 @@ To enable this configuration use the `extends` property in your
 }
 ```
 
+## Configuration
+
+There are some configuration options available that will be shared across all the plugin rules. This is achieved using [ESLint Shared Settings](https://eslint.org/docs/user-guide/configuring/configuration-files#adding-shared-settings). These shared settings are meant to be used if you need to restrict the Aggressive Reporting mechanism, **so please before configuring any of these settings**, read more about [`eslint-plugin-testing-library` Aggressive Reporting mechanism](docs/migrating-to-v4-guide.md#aggressive-reporting), and [how it's affected by these settings](docs/migrating-to-v4-guide.md#shared-settings).
```

Comment:
I think it'd be valuable to call out here that restricting Aggressive Reporting is a specialized/advanced feature. The name makes it seem bad and scary and might make people think they need to configure this. In reality, this is not needed for most people to ever touch.